recipes/yaml-cpp4rkt/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set version = "0.6.4.2.rkt" %}
```

I would rather keep the version numeric, you already have `rkt` in the name
```suggestion
{% set version = "0.6.4.2" %}
```